tests/python/test_with_dask.py

```diff
@@ -482,16 +481,59 @@ def test_predict():
             assert pred.ndim == 1
             assert pred.shape[0] == kRows
 
-            margin = xgb.dask.predict(client, model=booster, data=dtrain, output_margin=True)
+            margin = xgb.dask.predict(client, model=booster, data=dtrain,
+                                      output_margin=True)
             assert margin.ndim == 1
             assert margin.shape[0] == kRows
 
-            shap = xgb.dask.predict(client, model=booster, data=dtrain, pred_contribs=True)
+            shap = xgb.dask.predict(client, model=booster, data=dtrain,
+                                    pred_contribs=True)
             assert shap.ndim == 2
             assert shap.shape[0] == kRows
             assert shap.shape[1] == kCols + 1
 
 
+def test_aft_survival():
+    # survival doesn't handle empty dataset well.
```

Comment:
I did not know this. How can we fix this?

Reply:
@hcho3 There are checks:
``` c++
    CHECK_NE(info.labels_lower_bound_.Size(), 0U)
      << "labels_lower_bound cannot be empty";
    CHECK_NE(info.labels_upper_bound_.Size(), 0U)
      << "labels_upper_bound cannot be empty";
```

in `survival_metric.cu::EvalEWiseSurvivalBase::Eval`.

Reply:
Should we remove the checks and have the objective return zero?

Reply:
Depends on the synchronization I think.

Reply:
AFT is an element-wise objective, so it should behave like `reg:squarederror` when it comes to handling empty datasets. Let me create a new issue for this.

Reply:
Opened #6139